### PR TITLE
Issue #3190: Convert SupportUtils to Kotlin

### PR DESCRIPTION
**Pull request by SubhrajyotiSen**

### CODE REVIEW

File: app/src/main/java/org/mozilla/focus/utils/SupportUtils.kt [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+import org.mozilla.focus.locale.Locales
+import org.mozilla.focus.session.SessionManager
+import org.mozilla.focus.session.Source
+
+import java.io.UnsupportedEncodingException
+import java.net.URLEncoder
+import java.util.Locale
+
+object SupportUtils {
+    val HELP_URL = "https://support.mozilla.org/kb/what-firefox-focus-android"
```
Comment (colintheshots) [inline app/src/main/java/org/mozilla/focus/utils/SupportUtils.kt]: These can be "const val". https://kotlinlang.org/docs/reference/properties.html#compile-time-constants

**Comment by colintheshots** [comment]
It looks like @pocmo made an overlapping PR. We do always appreciate your help!

**Comment by pocmo** [comment]
Oh, sorry, I should have looked at the open PRs first.

**Comment by SubhrajyotiSen** [comment]
Closing since this overlaps with #3299

**Pull request closed without merging**